client/src/components/settings/About.js

```diff
@@ -105,6 +111,14 @@ class AboutSettings extends Component {
 
   handlePictureChange = e => {
     const value = e.target.value.slice(0);
+    var testImage = new Image();
+    testImage.src = value;
+    testImage.addEventListener('load', () => {
+      this.setState({ isPictureUrlValid: true });
+    });
+    testImage.addEventListener('error', () => {
+      this.setState({ isPictureUrlValid: false });
+    });
```

Comment:
How can we make these mutually exclusive

Reply:
Is there any situation where they can happen at the same time ?

Reply:
If not, then why use two handlers? Why not set the state to be false and depend only the `load` to update the state to be truthy.

Reply:
The Image object needs to listen to the 2 events so that if one enters a faulty url and then correct it, he can submit it (because the load event will be fired and then the state will change), and to also prevent one from entering first a correct url and saving it and then editing it to become faulty (the error event will be fired). The user should not be able to submit it.

If we set the state to be false and we keep only the load event, I can first enter a correct url, submit it (the state will become true), come back to the form and edit it to become incorrect, I will still have the possibility to submit